src/external/curve/ICurveMinter.sol

```diff
@@ -2,7 +2,9 @@
 pragma solidity 0.6.12;
 
 interface ICurveMinter {
-    function get_virtual_price() external view returns (uint);
-    function lp_price() external view returns (uint);
-    function price_oracle() external view returns (uint);
+    function get_virtual_price() external view returns (uint256);
```

Comment:
`uint` to `uint256` generates the same bytecode but a different AST (https://github.com/prettier-solidity/prettier-plugin-solidity#explicit-types)